js/packages/components/inputs/Input.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import { storiesOf } from '@storybook/react-native'
+import React, { useState } from 'react'
+import { ScrollView, Text, View } from 'react-native'
+
+import {
+	MediumInput,
+	MediumClearableInput,
+	LargeInputWithIcon,
+	LargeInput,
+	SmallInput,
+	SmallClearableInput,
+} from '.'
+import { AppDecorator } from '../../../.storybook/preview'
+
+const Spacer = () => <View style={{ height: 16 }} />
```

Comment:
Same here, can you add styles to a `Stylesheet` variable at the bottom of the file ?